domain/src/main/assets/MjZzEVOG47_1.json

```diff
@@ -2694,7 +2694,7 @@
               "dest": "Matthew gets conned",
               "feedback": {
                 "content_id": "feedback_1",
-                "html": "<p>Yes, that's right -- congratulations, you figured out Crumb's trick!</p><p>Let's see what happens to Matthew, though...</p>"
+                "html": " "
```

Comment:
Nit: prefer empty string for consistency with the textproto version.

That being said, I think it's preferable to set the feedback to null here & absent for textproto (since that's what the backend would generate for such a case). You'll also need to remove any translations or voiceovers corresponding to feedback_1 for this state.

Reply:
If I set `"feedback": null`, all tests break. 
I don't think it's necessary to remove all the voiceover and translation, other test depends on those. I think it's fine to have a nonsensical proto for testing purposes, since all we care about is html content?

Reply:
Since you're keeping the feedback, we should also keep the translation and voiceover. It's odd that null breaks it; I suppose that just means that the JSON reader for states isn't robust against this case. Either way, I think leaving this as-is is fine. Thanks for checking!